tests/actions/test_templatetags.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime, timezone
+
+import pytest
+
+import actions.templatetags.resolve_links as resolve_links
+from actions.models import Action
+
+
+@pytest.fixture
+def version():
+    action = Action.objects.create(
+        org="opensafely-actions",
+        repo_name="test-action",
+    )
+    return action.versions.create(
+        tag="v1.0",
+        committed_at=datetime(2024, 1, 1, tzinfo=timezone.utc),
+    )
+
+
+def test_resolve_relative_links_with_text_links(version):
+    html = (
+        'Here is link 1\n<a href="link_1.md">Link 1</a>\n'
+        'Here is link 2\n<a href="https://site.com">Link 2</a>'
+    )
+    readme = resolve_links.resolve_relative_links(html, version)
+
+    assert readme == (
+        "Here is link 1\n"
+        '<a href="https://github.com/opensafely-actions/test-action/blob/v1.0/link_1.md">Link 1</a>\n'
+        "Here is link 2\n"
+        '<a href="https://site.com">Link 2</a>'
+    )
+
+
+def test_resolve_relative_links_with_image_links(version):
```

Comment:
And again, let's be clearer with our terminology. We are testing that relative URLs in image tags are resolved to GitHub.